public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,106 @@
+# metrics diff
+
+Find and print [project metrics](/doc/command-reference/metrics#description)
+changes between commits, commit and a working tree, etc.
+
+## Synopsis
+
+```usage
+usage: dvc metrics diff [-h] [-q | -v] [--targets [TARGETS [TARGETS ...]]]
+                        [-t TYPE] [-x XPATH] [-R] [--show-json]
+                        [a_ref] [b_ref]
+
+positional arguments:
+  a_ref                 Git reference from which diff is calculated. If
+                        omitted `HEAD`(latest commit) is used.
+  b_ref                 Git reference to which diff is calculated. If omitted
+                        current working tree is used.
```

Comment:
Interesting @efiop @dmpetrov so this is not like `dvc diff`, where `a_ref` is required and `b_ref` defaults to `HEAD`. `dvc diff` also doesn't have the ability to check the `working tree` AFAIU.

Should we review `dvc diff` so it behaves this way as well? (I can open a separate issue.)

Reply:
p.s. took me a while to understand this part correctly because I was naturally assuming it works like `dvc diff`. So this will probably create confusion for users as well.

Reply:
@jorgeorpinel `dvc diff` is under reconstruction and will also not require `a_ref`, same as `git diff`.

Reply:
Cool. Can you link to the core repo issue/PR please?

Reply:
Is this the one? #2999

Reply:
I mean iterative/dvc/issues/2982

Reply:
@jorgeorpinel Yes, correct.

Reply:
Thanks